src/drivers/fpga/xrt/lib/xleaf/test.c

```diff
@@ -167,14 +162,21 @@ static int xrt_test_remove(struct platform_device *pdev)
 static int
 xrt_test_leaf_ioctl(struct platform_device *pdev, u32 cmd, void *arg)
 {
+	int ret = 0;
+
 	switch (cmd) {
 	case XRT_XLEAF_EVENT:
 		xrt_test_event_cb(pdev, arg);
 		break;
+	case XRT_XLEAF_TEST_A:
+		ret = xrt_test_ioctl_cb_a(pdev, arg);
+		break;
+	case XRT_XLEAF_TEST_B:
+		ret = xrt_test_ioctl_cb_b(pdev, arg);
```

Comment:
checkpatch does not complain about a missing 'break' here? i'd suggest to add one so that "XRT_XLEAF_TEST_B" does not have to be right before "default" case.

Reply:
Fixed